next_gen_ui_demo/lib/common/shader_effect.dart

```diff
@@ -0,0 +1,359 @@
+// Copyright 2023 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' as ui;
+
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_animate/flutter_animate.dart';
+
+/**
+ * This is an unfinished, pre-release effect for Flutter Animate:
+ * https://pub.dev/packages/flutter_animate
+ *
+ * It includes a copy of `AnimatedSampler` from Flutter Shaders:
+ * https://github.com/jonahwilliams/flutter_shaders
```

Comment:
Are we still waiting for a new version to be published?

Reply:
Yup. It'll probably land in the API once Jonah is happy.